### ISSUE TITLE
Question about npy file generated by preprocess_data.py

### ISSUE THREAD

**Issue by WangZYZZ** [limhoyeon/ToothGroupNetwork#7]
I try to use preprocess_data.py to generated processed npy file. But the data in npy file makes me confused. 
[[ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]
 [ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]
 [ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]
 ...
 [ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]
 [ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]
 [ 0.2485581  -0.29567248  0.1667352  -0.85212744  0.42836872  0.30063109]]

[2. 2. 2. ... 2. 2. 2.]

I think for each npy file, there should be different coordinates and different labels. But for the npy file above, only label 2 exists and all coordianates are the same. How can i solve it ?

**Comment by WangZYZZ** [comment]
And I met an error when running command of 'train the Farthest Point Sampling model'
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/8e314fc8-a91d-440a-a5b7-80c6c1953206)

**Comment by limhoyeon** [comment]
maybe this issue is related to https://github.com/limhoyeon/ToothGroupNetwork/issues/4.
there are issues with CUDA functions not executing properly in some PC environments. I plan to remove the CUDA functions and replace them with Python code. Please bear with me a little longer, or alternatively, you can try reinstalling the 'pointops' library from https://github.com/POSTECH-CVLab/point-transformer/tree/master/lib/pointops.

**Comment by WangZYZZ** [comment]
Before I run 'labeled_vertices = gu.resample_pcd([labeled_vertices], 24000, "fps")[0]'  in preprocess_data.py, the labeled_vertices is like 
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/812dce03-b5f3-4936-bbe3-f874cffc0fb6)
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/8bfa24a6-bab7-4cef-aae2-548df1a84494)
The data seems to be ok.
But after using  'labeled_vertices = gu.resample_pcd([labeled_vertices], 24000, "fps")[0]', the labeled_vertices is like
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/c6d9c229-8a45-4ef6-aa87-d8a3588cee3e)
So is it the problem of gu.resample_pcd ?

**Comment by WangZYZZ** [comment]
For the pointops, I remeber I have installed it using the setup.py
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/d7f7aebf-60a2-4c41-a7e1-136d3c0fc8a4)

**Comment by WangZYZZ** [comment]
should I change anything of the def resample_pcd ?
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/3dc54f42-4d22-4710-986c-07e13740da1a)

**Comment by WangZYZZ** [comment]
Hi, I think i have solved the problem after reinstalling the pointops from your URL.
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/74094186-8663-443a-a4bc-e04eafd2f9b3)
May I know why the pointops can cause this question ?

**Comment by limhoyeon** [comment]
It seems that I have made some wrong modifications in the PointOps library in this repository,,

**Comment by WangZYZZ** [comment]
Hi, I have started training the Farthest Point Sampling model from last Friday. But it is still running. May I know how much time did you use to train it ?
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/ce459dd1-4995-4e60-ae76-da8a2c8ec4be)

**Comment by limhoyeon** [comment]
Maybe it is hard to track experiments without wandb.

Using wandb, you can track the experiments, as shown below. I trained provided models for 60 epochs."
<img width="1030" alt="image" src="https://github.com/limhoyeon/ToothGroupNetwork/assets/70117866/e33b8dee-9a1d-4610-9a03-27e3295ddc96">

**Comment by WangZYZZ** [comment]
Hi, these are my wandb graphs. Is it right? I think there are redundant steps because the lines are almost horizontal after 60 steps. May I know where i can change the number of step or epoch?
![image](https://github.com/limhoyeon/ToothGroupNetwork/assets/116799416/4cb32c13-2c76-4d2a-99a3-a9a77b0a5f69)

**Comment by WangZYZZ** [comment]
Does the running of this train require me to manually stop it? There is no progress bar on the training screen, and I don't see where to set the STEP number.

**Comment by limhoyeon** [comment]
I looked at wandb graph and manually stopped the train. 
you can change the scheduler step(in train_configs/modelname.py) and  change the learning rate for each epoch(step_lr graph in wandb).
